libero_raw_data_dir: /home-ssd/Users/nsgm_lx/xingyouguang/datasets/robotics/libero/libero_spatial
libero_raw_data_dir: /home-ssd/Users/nsgm_lx/xingyouguang/datasets/robotics/libero/libero_spatial
04/22 [22:29:57] INFO     | >> PyTorch version 2.2.0+cu121 available.                                                                                                                                config.py:54
                 INFO     | >> TensorFlow version 2.15.0 available.                                                                                                                                 config.py:112
Regenerating libero_spatial dataset!
v: 0.05->-5.0, 0.15->5.0, c: 0.25->0.25, 0.25->0.25
[info] using task orders [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
Creating LeRobotDataset at /home-ssd/Users/nsgm_lx/xingyouguang/datasets/robotics/libero/libero_spatial_no_noops_island_split_lerobot/xyg_20_10_-10.0_90.0/v-0.050-0.150_0,1,3,5,8
Processing 5 tasks in parallel
[info] using task orders [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
current task id: 5, pid: 4087432
[Warning]: datasets path /home-ssd/Users/nsgm_lx/xiejunlin/LIBERO/libero/libero/../datasets does not exist!
Processing task 5: pick_up_the_black_bowl_on_the_ramekin_and_place_it_on_the_plate
[info] using task orders [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
current task id: 8, pid: 4087433
[Warning]: datasets path /home-ssd/Users/nsgm_lx/xiejunlin/LIBERO/libero/libero/../datasets does not exist!
Processing task 8: pick_up_the_black_bowl_next_to_the_plate_and_place_it_on_the_plate
[info] using task orders [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
current task id: 3, pid: 4087431
[Warning]: datasets path /home-ssd/Users/nsgm_lx/xiejunlin/LIBERO/libero/libero/../datasets does not exist!
Processing task 3: pick_up_the_black_bowl_on_the_cookie_box_and_place_it_on_the_plate
[info] using task orders [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
current task id: 1, pid: 4087430
[Warning]: datasets path /home-ssd/Users/nsgm_lx/xiejunlin/LIBERO/libero/libero/../datasets does not exist!
Processing task 1: pick_up_the_black_bowl_next_to_the_ramekin_and_place_it_on_the_plate
[info] using task orders [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
current task id: 0, pid: 4087429
[Warning]: datasets path /home-ssd/Users/nsgm_lx/xiejunlin/LIBERO/libero/libero/../datasets does not exist!
Processing task 0: pick_up_the_black_bowl_between_the_plate_and_the_ramekin_and_place_it_on_the_plate
